MParT/Utilities/ArrayConversions.h

```diff
@@ -320,6 +320,28 @@ namespace mpart{
 
     }
 
+    template<typename DeviceMemoryType, typename... OtherTraits>
```

Comment:
I don't think this function is needed anymore

Reply:
oops, I forgot to add that file to the commit